src/index.ts

```diff
@@ -0,0 +1,78 @@
+import * as capitano from 'capitano';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+import JSZip from 'jszip';
+import * as Path from 'path';
+import * as skhema from 'skhema';
+
+const specPath: string = '/source/specification/spec.json';
+const zip =  new JSZip();
+
+const generate = async (
+	params: { folder: string },
+	options: { output: string },
+) => {
+
+	const rfq = await validateSpec(params.folder + specPath); // generate rfq json from spec.json and schema
+	zip.file('rfq.json', JSON.stringify(rfq, null, 2)); // add rfq.json to the release
+
+	const testFile = fs.readFileSync(Path.join(params.folder, 'testing', 'Testing.md'));
+	const testContent = Buffer.from(testFile).toString('utf8');
+	zip.file('Testing.md', testContent);
+
+	packageFiles(Path.join(params.folder, 'outputs'));
+
+	const data = await zip.generateAsync({
+		type: 'nodebuffer',
+		compression: 'DEFLATE'
+	});
+	fs.writeFileSync(Path.join(options.output, `release.zip`), data, 'binary');
+};
+
+// check the spec file against schema for the project type - this will have to change if we get rid of the spec.json file from the repos
```

Comment:
@rcooke-warwick I can't remember, but have we discussed doing something like this (removing the spec.json)? Can't seem to think of why we'd want that

Reply:
@anujdeshpande at the moment we were using spec.json to hold parameters not in the source file, like thickness/material. We talked about maybe having a file per manufacturing run instead though but we didn't decide on anything